### DISCUSSION TITLE
sql parser error when using mssql FOR JSON PATH

### DISCUSSION THREAD

**Discussion by Pieter3033** [sqlpage/SQLPage#126] [Q&A] [answered]
Hi,

The newest form component is really cool 🔥.

I am currently struggling to populate the `options `of a `select type` in the `form` component.
I am using MSSQL.

I have attempted the following: 
`(SELECT Column1 AS label, Column2 AS value FROM MyTable FOR JSON PATH) AS options`

Error:
![image](https://github.com/lovasoa/SQLpage/assets/141907966/ef20a898-cc6e-4b81-8597-2035032b9076)

Note, I have tested this SQL statement inside of MSSQL and the output is as expected.
Maybe I am struggling with the syntax?

Regards

**Comment by lovasoa** [comment, marked as answer]
Hello ! This looks like an issue in the SQLPage SQL parser, which does not support the non-standard Microsoft SQL syntax for json.

Let's move the discussion to this issue: https://github.com/lovasoa/SQLpage/issues/127

I'll provide a workaround to use until the underlying parsing issue is fixed.
